pkg/support/runsupportbundle_test.go

```diff
@@ -250,3 +261,13 @@ func givenWriter(t *testing.T) *mocks.MockFileWriter {
 	ctrl := gomock.NewController(t)
 	return mocks.NewMockFileWriter(ctrl)
 }
+
+func givenProvider(t *testing.T) *providerMocks.MockProvider {
+	ctrl := gomock.NewController(t)
+	return providerMocks.NewMockProvider(ctrl)
+}
+
+func machineConfigs() []providers.MachineConfig {
```

Comment:
this test needs to be improved, and actually test that the correct collectors are being called.